lib/tasks/emails.rake

```diff
@@ -0,0 +1,20 @@
+namespace :metamaps do
+  desc "delivers recent map activity digest emails to users"
+  task deliver_map_activity_emails: :environment do
+    summarize_map_activity
+  end
+
+  def summarize_map_activity
+    Map.all.find_each do |map|
+      map.followers.each do |user|
+        # add logging and rescue-ing
+        # and a notification of failure
+        next unless MapPolicy.new(user, map).show? # just in case the permission changed
+        next unless user.emails_allowed
```

Comment:
To make it more granular? Yeah, now would be a good time. I have ideas abt it I'll work on something